modules/dockermodelrunner/internal/sdk/client/pull.go

```diff
@@ -0,0 +1,37 @@
+package client
+
+import (
+	"context"
+	"fmt"
+	"log"
+	"net/http"
+	"strings"
+)
+
+// PullModel creates a model in the Docker Model Runner, by pulling the model from Docker Hub.
+func (c *Client) PullModel(ctx context.Context, fqmn string) error {
+	payload := fmt.Sprintf(`{"from": %q}`, fqmn)
+	reqURL := c.baseURL + "/models/create"
+
+	req, err := http.NewRequestWithContext(ctx, http.MethodPost, reqURL, strings.NewReader(payload))
+	if err != nil {
+		return fmt.Errorf("new post request (%s): %w", reqURL, err)
+	}
+
+	log.Default().Printf("🙏 Pulling model %s. Please be patient, no progress bar yet!", fqmn)
```

Comment:
question: do we want to add a TODO to remove these log lines when we have a better solution?

Reply:
Working on a progress bar atm, using https://github.com/schollz/progressbar/ as reference

Reply:
I added a progress bar, so this is not valid anymore